sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -392,6 +417,69 @@ describe("RequestResponseLink", function() {
         `Incorrect error received "${err.message}"`
       );
     }
+    assertItemsLengthInResponsesMap(link["_responsesMap"], 0);
+  });
+
+  it("should abort a request and response correctly when abort signal is fired after sometime", async function() {
+    const connectionStub = stub(new Connection());
+    const rcvr = new EventEmitter();
+    let req: any = {};
+    connectionStub.createSession.resolves({
```

Comment:
Hey, this is nice. Did not know about 'resolves'!

Reply:
That's `sinon`!